src/components/navigation/UserAccounts.js

```diff
@@ -38,24 +48,54 @@ const Account = styled.div`
         color: black;
     }
 
+    .balance {
+        color: #00C08B;
+        font-weight: 400;
+    }
+
     &.additional-account {
         :last-of-type {
             border-bottom: 0;
         }
+
+        .refresh {
+            position: absolute;
+            width: 20px;
+            height: 100%;
```

Comment:
Nice addition to be able to 'refresh' the account balance. Can you please also center the icon vertically (once @corwinharrell has provided a new icon)

Reply:
Also, very much a nit, but could skip a lot of the style (and absolute positioning) if you make the icon a component and use flexbox to position the elements within the `<Account>`, similar to how it's done here: https://github.com/near/near-wallet/blob/master/src/components/navigation/NavLinks.js#L79

Reply:
Sure. I'm actually not sure about this design, just added it there to make it work - but maybe, the icon should be added just after the balance amount? @corwinharrell maybe you can propose something nice here?